charts/opensearch-dashboards/CHANGELOG.md

```diff
@@ -13,6 +13,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 ### Security
 ---
+## [2.17.0]
```

Comment:
Hey @gaiksaya can you update the change log at bottom of this file?

Reply:
^^

Reply:
Done

Reply:
Looks like [2.16.1] was missed in past PR.